File: Baekjoon_Online_Judge/NO_10987/10987.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 08                 | 
 | Day 02                   | 
 | 10987 모음의 개수          | 
 ---------------------------- 
*/

int main(){
    string word;
    int cnt = 0;

    cin >> word;

    for(auto ch: word){
        if(ch =='a' or ch =='e' or ch =='i' or ch =='o' or ch =='u') cnt += 1;
    }

    cout << cnt;
    return 0;
}

File: Baekjoon_Online_Judge/NO_10987/10987.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 08                 | #
# | Day 02                   | #
# | 10987 모음의 개수          | #
# ---------------------------- #


if __name__ == "__main__":
    word = r_input().rstrip()
    res = 0

    for ch in word:
        if ch in 'aeiou':
            res += 1

    print(res)
